chain/store/store.go

```diff
@@ -765,15 +765,15 @@ func (cs *ChainStore) TryFillTipSet(ts *types.TipSet) (*FullTipSet, error) {
 	return NewFullTipSet(out), nil
 }
 
-func (cs *ChainStore) GetRandomness(ctx context.Context, blks []cid.Cid, tickets []*types.Ticket, lb int64) ([]byte, error) {
+func (cs *ChainStore) GetRandomness(ctx context.Context, blks []cid.Cid, tickets []*types.Ticket, lb uint64) ([]byte, error) {
 	ctx, span := trace.StartSpan(ctx, "store.GetRandomness")
 	defer span.End()
-	span.AddAttributes(trace.Int64Attribute("lb", lb))
+	span.AddAttributes(trace.Int64Attribute("lb", int64(lb)))
 
-	if lb < 0 {
-		return nil, fmt.Errorf("negative lookback parameters are not valid (got %d)", lb)
-	}
-	lt := int64(len(tickets))
+	// if lb < 0 {
+	// 	return nil, fmt.Errorf("negative lookback parameters are not valid (got %d)", lb)
```

Comment:
maybe check if `lb > math.MaxUint32`. That would imply someone did math wrong and the number underflowed

Reply:
added